redwood-treehouse-host/src/androidUnitTest/kotlin/app/cash/redwood/treehouse/TreehouseLayoutTest.kt

```diff
@@ -50,7 +50,8 @@ class TreehouseLayoutTest {
   private val context = RuntimeEnvironment.getApplication()!!
```

Comment:
This can be eliminated. The `activity` within each test can be used as the first argument to the `TreehouseLayout` constructor.